src/app/learn/reference/configuration/index.html

```diff
@@ -102,7 +102,7 @@
     </li>
     <li>
       <p>
-        <sky-code>title</sky-code> — Controls the page title at the template level for the <sky-code>skyux serve</sky-code> command while waiting for the application to load. After it loads, you can use <a href="https://angular.io/docs/ts/latest/cookbook/set-document-title.html">the Angular <sky-code>Title</sky-code> service</a> to set the page title. The default is <sky-code>Blackbaud — SKY UX Application</sky-code>, so if you do not want this default to appear in the title bar while the application loads, use the <sky-code>title</sky-code> to provide your preferred title.
+        <sky-code>title</sky-code> — Controls the page title at the template level when you run <sky-code>skyux serve</sky-code> to serve a SPA. To control your SPA's page title outside of serving locally, use the <a href="https://angular.io/docs/ts/latest/cookbook/set-document-title.html">Angular <sky-code>Title</sky-code> service</a>. For Blackbaud developers, <sky-code>SkyAppTitleService</sky-code> is the preferred method to <a href="https://docs.blackbaud.com/engineering-system-docs/learn/spa/spa-title-service">set the page title in a SPA</a>. When you set page titles, follow the recommended format in the <a routerLink="/design/guidelines/page-layouts" fragment="page-titles">page design guidelines</a>.
```

Comment:
The first sentence still isn't quite accurate. If I were to describe the `title` property, I'd say something like "Specifies the page title on the HTML file that is emitted during `skyux build` and served when running `skyux serve -local`. This title is displayed while the application loads. After it loads..." etc. etc., and at some point I'd mention "The `title` property has no effect when running `skyux serve` since the SKY UX Host service controls the SPA's page title while the SPA is loaded. "

Reply:
Just pushed up changes to address the feedback. Let me know if I got anything wrong or if you think we should change anything else. Thanks!